testsuite/testsuite.c

```diff
@@ -102,7 +102,9 @@ int testsuite_test(int argc, char** argv)
     func_args server_args;
 
     tcp_ready ready;
+#if !defined(NETOS)
```

Comment:
A few lines down, the variable serverThreadStackContext is also only used when !defined(NETOS).

Reply:
Fixed